vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -223,29 +196,41 @@
-            		if(!chooseFromExisting){
-            			reader.onload = function (theFile) 
-            			{        	
-            			var image = new Image();
-            			image.src = theFile.target.result;
-            			image.onload = function() 
-            			{
-            				imageblock = createImageBlock(reader, this.width);  
-            				$('#slideSpace').append(imageblock); 
-            				$(imageblock[0]).mouseenter(onMouseEnter).mouseleave(onMouseLeave).dblclick(onDoubleClick);
-            			};          
-            			} 
-            		}
-            		else
-            		{
-            			var image = new Image();
-            			image.src = $('#selectedImage').find('img').attr('src');
-            			formData.append('imageId',imageId);
-            			imageblock = createImageBlockWithExistingImage($('#selectedImage').find('img').attr('src'),image.width);  
-            			$('#slideSpace').append(imageblock); 
-            			$(imageblock[0]).mouseenter(onMouseEnter).mouseleave(onMouseLeave).dblclick(onDoubleClick);
-            		}
+
+            	    var fileInput = document.getElementById('file'); // Assuming you have a file input element with id 'fileInput'
+            	    var url = "[(@{'/staff/module/'+${module.id}+'/slide/'+${slide.id}+'/image/?'+${_csrf.parameterName}+'='+${_csrf.token}})]";
+            	    if (fileInput.files.length === 0) {
+            	    	$('#chooseFromLocal').trigger( "click" );
+                    	// Initialize selected image ID to blank, on clicking cancel button
+                    	$("#uploadImage").data('value', '');
+                        $("#addImgAlert").hide();
+                        $(".open").removeClass("open");
+            	    }
+            	    else{
+            	    	
+                		if(!chooseFromExisting){
+                			reader.onload = function (theFile) 
+                			{        	
+                			var image = new Image();
+                			image.src = theFile.target.result;
+                			image.onload = function() 
+                			{
+                				imageblock = createImageBlock(reader, this.width);  
+                				$('#slideSpace').append(imageblock); 
+                				$(imageblock[0]).mouseenter(onMouseEnter).mouseleave(onMouseLeave).dblclick(onDoubleClick);
+                			};          
+                			} 
+                		}
+                		else
+                		{
+                			var image = new Image();
+                			image.src = $('#selectedImage').find('img').attr('src');
+                			formData.append('imageId',imageId);
+                			imageblock = createImageBlockWithExistingImage($('#selectedImage').find('img').attr('src'),image.width);  
+                			$('#slideSpace').append(imageblock); 
+                			$(imageblock[0]).mouseenter(onMouseEnter).mouseleave(onMouseLeave).dblclick(onDoubleClick);
+                		}
```

Comment:
indentation of all of the above

Reply:
Reindented everything according to the 4 tab space indentation convention, which was out of place in even in the develop branch's slide.html file